docs/source/specs/openapi.json

```diff
@@ -206,6 +225,25 @@
         ],
         "summary": "List the groups for a tenant",
         "operationId": "listGroups",
+        "parameters": [{
+            "name": "page",
+            "in": "query",
+            "description": "Parameter for selecting the page of data (default is 1).",
+            "required": false,
+            "schema": {
+              "type": "integer"
+            }
+          },
+          {
+            "name": "page_size",
+            "in": "query",
+            "description": "Parameter for selecting the amount of data in a page (default is 10).",
+            "required": false,
+            "schema": {
+              "type": "integer"
```

Comment:
@chambridge you should be able to use default and minimum here
https://github.com/ManageIQ/service_portal-api/blob/d6b58bfdaedf928e3574d61211fe9cd7f0973284/public/service_portal/v0.1.0/openapi.json#L825